### PR TITLE
Add docs for enable-job-log-tmpfile

### PR DESCRIPTION
**Pull request by chrisdothtml**
Adding docs for the feature added in https://github.com/buildkite/agent/pull/1564

Preview:
**Agent configuration**
Screen Shot 2022-07-29 at 1 32 49 PM

**Environment variables**
Screen Shot 2022-07-29 at 1 32 36 PM

### CODE REVIEW

**Comment by pzeballos** [comment]
Hey @karensawrey! This one is good to go after you check the wording 😎

File: data/agent_attributes.yaml [hunk 1]
```diff
@@ -53,6 +53,12 @@ attributes:
   required: false
   desc: |
       The number of idle seconds to wait before shutting down. When setting this option alongside `spawn`, all agents started by `spawn` must be idle before the timeout counter will begin. After the timeout, the spawned agents will all disconnect.
+- name: enable-job-log-tmpfile
+  env_var: BUILDKITE_ENABLE_JOB_LOG_TMPFILE
+  default_value: "false"
+  required: false
+  desc: |
+      Enables the writing of job logs into a temporary file which can be accessed via the `BUILDKITE_JOB_LOG_TMPFILE` environment variable. Must be using Agent v3.36.0 or above.
```
Comment (chrisdothtml) [inline data/agent_attributes.yaml:61]:
```suggestion
      Enables the writing of job logs into a temporary file which can be accessed via the `BUILDKITE_JOB_LOG_TMPFILE` environment variable. Introduced in [v3.36](https://github.com/buildkite/agent/releases/tag/v3.36.0).
```

Could possibly update this wording to resemble the existing wording [here](https://buildkite.com/docs/agent/v3/configuration#redacted-vars:~:text=Introduced%20in%20v3.31.)

**Review by plaindocs — COMMENTED** [review]
Nice work @chrisdothtml 

We should probably :-p add the environment variable to the docs also.

**Comment by chrisdothtml** [comment]
@plaindocs just added to the environment vars page as well

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
Thanks again @chrisdothtml :-)

**Pull request merged**